File: OLSource1/v1/VS_officedev/codesnippet/VisualBasic/walkthrough--binding-content-controls-to-custom-xml-parts_2.vb
    Private Function GetXmlFromResource() As String
        Dim asm As System.Reflection.Assembly = _
            System.Reflection.Assembly.GetExecutingAssembly()
        Dim stream1 As System.IO.Stream = asm.GetManifestResourceStream( _
                "EmployeeControls.employees.xml")

        Using resourceReader As System.IO.StreamReader = _
            New System.IO.StreamReader(stream1)
            If resourceReader IsNot Nothing Then
                Return resourceReader.ReadToEnd()
            End If
        End Using

        Return Nothing
    End Function

File: OLSource1/v1/VS_officedev/codesnippet/CSharp/walkthrough--binding-content-controls-to-custom-xml-parts_2.cs
        private string GetXmlFromResource()
        {
            System.Reflection.Assembly asm =
                System.Reflection.Assembly.GetExecutingAssembly();
            System.IO.Stream stream1 = asm.GetManifestResourceStream(
                "EmployeeControls.employees.xml");

            using (System.IO.StreamReader resourceReader =
                    new System.IO.StreamReader(stream1))
            {
                if (resourceReader != null)
                {
                    return resourceReader.ReadToEnd();
                }
            }

            return null;
        }